percentage_deception: 1